_rules/explicit-SVG-image-has-name-7d6734.md

```diff
@@ -0,0 +1,179 @@
+---
+id: 7d6734
+name: '`svg` element with explicit role has accessible name'
```

Comment:
The PR title "explicit SVG image [...]" may have unintended associations, I think this name "explicit role" is much better.